durables.conf.example

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Why are `precision` and `nai` defined?

Is `creator` really something that can be controlled here, or is it's value effectively edict-ed to us by other tools?

Reply:
Amounts are always stored as satoshis, but the precision of the satoshi depends on the asset type (nai).  Thus, the asset object amount is dependant on precision and nai.  I would prefer to express the asset this way rather than a string like "2000.000 TESTS" because it allows us to avoid extra configuration requirements.

I haven't tested using another creator, but it is currently coded to allow that flexibility.

Reply:
### precision & nai

Hm. So `precision` is a function of the `nai`, not the amount? 

To use the 1st two items in the example conf - would the precision of the 2nd item be ok (and have the effect of giving a full 250 of the coin represented by `@@000000021` ?

```json
"accounts": [
    {
      "name": "tn-one",
      "vesting": {"amount": "250000000", "precision": 3, "nai": "@@000000021"},
      "creator": "initminer"
    }, {
      "name": "tn-two",
      "vesting": {"amount": "250", "precision": 0, "nai": "@@000000021"},
      "creator": "initminer"
    }

....
```

### creator

>I haven't tested using another creator, but it...
By **it** do you mean the entire account creation stack from here to where the debug plugin  causes the accounts to be created?

Reply:
You can't change the precision of an nai.

https://github.com/steemit/steem/blob/807fb40ec137a987dc53cee6d8455c7b6c47aeed/libraries/protocol/asset.cpp#L190

I recommend sticking to the fully canonoical asset object.

---

I just mean you should be able to set the creator to any valid account that has the funds to pay the fee.  I have tested `initminer` because that account has the funds at bootstrap.

Reply:
Looking at `https://developers.steem.io/apidefinitions/#apidefinitions-broadcast-ops-comment` I see that precision is present in our api as well. I wonder if this is some kind of extremely poor validity check.

Given that this structure appears to be canon, I'm inclined to agree with you. 

Once merged we should test a non `initminer` account asap.

Reply:
I tested using `porter` as the creator on sandbox, and it worked.